src/egon/data/datasets/electricity_demand_timeseries/hh_profiles.py

```diff
@@ -195,28 +196,50 @@ class EgonEtragoElectricityHouseholds(Base):
 
 class HouseholdDemands(Dataset):
     def __init__(self, dependencies):
-        mv_hh_electricity_load_2035 = PythonOperator(
-            task_id="MV-hh-electricity-load-2035",
-            python_callable=mv_grid_district_HH_electricity_load,
-            op_args=["eGon2035", 2035],
-            op_kwargs={"drop_table": True},
-        )
+        tasks = (houseprofiles_in_census_cells,)
+
+        if (
+            "status2019"
+            in egon.data.config.settings()["egon-data"]["--scenarios"]
+        ):
+            mv_hh_electricity_load_2035 = PythonOperator(
+                task_id="MV-hh-electricity-load-2019",
+                python_callable=mv_grid_district_HH_electricity_load,
+                op_args=["status2019", 2019],
+                op_kwargs={"drop_table": True},
+            )
 
-        mv_hh_electricity_load_2050 = PythonOperator(
-            task_id="MV-hh-electricity-load-2050",
-            python_callable=mv_grid_district_HH_electricity_load,
-            op_args=["eGon100RE", 2050],
-        )
+            tasks = tasks + (mv_hh_electricity_load_2035,)
+
+        if (
+            "eGon2035"
+            in egon.data.config.settings()["egon-data"]["--scenarios"]
+        ):
+            mv_hh_electricity_load_2035 = PythonOperator(
+                task_id="MV-hh-electricity-load-2035",
+                python_callable=mv_grid_district_HH_electricity_load,
+                op_args=["eGon2035", 2035],
+            )
+
+            tasks = tasks + (mv_hh_electricity_load_2035,)
+
+        if (
+            "eGon100RE"
+            in egon.data.config.settings()["egon-data"]["--scenarios"]
+        ):
+            mv_hh_electricity_load_2050 = PythonOperator(
+                task_id="MV-hh-electricity-load-2050",
+                python_callable=mv_grid_district_HH_electricity_load,
+                op_args=["eGon100RE", 2050],
+            )
+
+            tasks = tasks + (mv_hh_electricity_load_2050,)
```

Comment:
wouldn't it be slightly cleaner, to use `elif` for the subsequent options?

Reply:
I think I have to use `if` because it is possible that more than one scenario name is in the list. If I would use `elif` only the first one would be executed.